etest/qemu.py

```diff
@@ -0,0 +1,34 @@
+"""QEMU interpreter management."""
+# Copyright (C) 2014 by Alex Brandt <alunduil@alunduil.com>
+#
+# etest is freely distributable under the terms of an MIT-style license.
+# See COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from etest import docker
+
+
+class qemu:
+    """QEMU interpreter management."""
+
+    def __init__(self, arch: str):
+        """Initialize."""
+        if arch != "amd64":
+            self.enabled = True
+
+    def __enter__(self):
+        """Start the QEMU interpreter."""
+        if self.enabled:
+            docker.pull("multiarch/qemu-user-static:latest")
+
+            self.container = docker.container.create_simple(
+                image="multiarch/qemu-user-static",
+                privileged=True,
+                command="--reset -p yes",
```

Comment:
This looks more like args than command, but I can't recall how Docker describes these things.

Reply:
This is actually a command. I was confused about it for a while, but its related to how the qemu interpreter receives them.

Reply:
Strange, but makes sense.  Perhaps a quick command with at least a link to the qemu documentation or something indicating this?

Reply:
I could add the qemu documentation to the top of the file as to make it easier for new devs

Reply:
That sounds like a good idea.  Having these things documented better than I managed last time will help immensely if interest again resurges.